website/docs/guides/container_service_extension_install.html.markdown

```diff
@@ -231,42 +261,30 @@ In order to create all the items listed above, the [proposed configuration][step
   Organization [Routed network][routed_network].
 - `tenant_routed_network_dns`: DNS Server for the Tenant Organization [Routed network][routed_network]. It can be left blank if it's not needed.
 
-If you wish to have a different networking setup, please modify the [proposed configuration][step2].
+If you wish to have a different networking setup, please modify the [step 2 configuration][step2].
 
-### CSE Server
+#### CSE Server
 
-There is also a set of resources created by the [proposed configuration][step2] that correspond to the CSE Server vApp.
-The generated VM makes use of the uploaded CSE OVA and some required guest properties.
+There is also a set of resources created by the [step 2 configuration][step2] that correspond to the CSE Server vApp.
+The generated VM makes use of the uploaded CSE OVA and some required guest properties:
 
-In order to do so, the [configuration][step2] asks for the following variables that you can customise in `terraform.tfvars`:
+- `cse_admin_username`: This must be the same CSE Administrator user created in the first step.
+- `cse_admin_password`: This must be the same CSE Administrator user's password created in the first step.
+- `cse_admin_api_token_file`: This specifies the path where the API token is saved and consumed.
 
-- `vcdkeconfig_template_filepath` references a local file that defines the `VCDKEConfig` [RDE][rde] contents. It should be a JSON template, like
-  [the one used in the configuration](https://github.com/vmware/terraform-provider-vcd/tree/main/examples/container-service-extension-4.0/entities/vcdkeconfig-template.json).
-  (Note: In `terraform.tfvars.example` the correct path is already provided).
-- `capvcd_version`: The version for CAPVCD. It should be "1.0.0" for CSE v4.0.
-- `capvcd_rde_version`: The version for the CAPVCD [RDE Type][rde_type]. It should be the same version used in Step 1.
-- `cpi_version`: The version for CPI. It should be "1.2.0" for CSE v4.0.
-- `csi_version`: The version for CSI. It should be "1.3.0" for CSE v4.0.
-- `github_personal_access_token`: Create this one [here](https://github.com/settings/tokens),
-  this will avoid installation errors caused by GitHub rate limiting, as the TKGm cluster creation process requires downloading
-  some Kubernetes components from GitHub.
-  The token should have the `public_repo` scope for classic tokens and `Public Repositories` for fine-grained tokens.
-- `cse_admin_user`: This should reference the CSE Administrator [User][user] that was created in Step 1.
-- `cse_admin_api_token`: This should be the API token that you created for the CSE Administrator after Step 1.
+#### UI plugin installation
 
-### UI plugin installation
+-> If the old CSE 3.x plugin is installed, you will need to remove it before installing the new one.
```

Comment:
For some it may be obvious, but for those in a hurry, I would add this, just to be extra clear:

```suggestion
-> If the old CSE 3.x UI plugin is installed, you will need to remove it before installing the new one.
```

Reply:
Added, thanks!